public/ts/member.ts

```diff
@@ -140,7 +140,10 @@ common.documentLoaded().then(() => {
                     </div>
                     <div>
                         <label  for='phone'           class="uk-form-label">Telefonnummer</label>
-                        <input name='phone'           class="uk-input readonly-input" value="${member.phone || ''}" disabled />
+                        <span style="width: 100%; display: flex;">
+                            <input name='phone'           class="uk-input readonly-input" value="${member.phone || ''}" disabled />
+                            <a href="/member/change_phone" class="uk-button uk-button-primary" >Byt</a>
```

Comment:
Jag tycker blå färg ger fel intryck eftersom man inte bör byta telefonnummer varje dag. Kanske kan sätta en varnande färg istället? :)

```suggestion
                            <a href="/member/change_phone" class="uk-button uk-button-danger" >Byt</a>
```

![image](https://user-images.githubusercontent.com/7572427/179400001-61ce9863-3cd4-4f49-8176-149d99bb77e8.png)
->
![image](https://user-images.githubusercontent.com/7572427/179400024-9229e7b0-7a37-4383-b61b-0e7cd6a180cf.png)